### PR TITLE
Update Quidel doc

### PR DESCRIPTION
**Pull request by jingjtang**
closes[ #838](https://github.com/cmu-delphi/delphi-epidata/issues/838)

**Prerequisites**:

- [ ] Unless it is a documentation hotfix it should be merged against the `dev` branch
- [ ] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [ ] Build is successful
- [ ] Code is cleaned up and formatted

### Summary

- Add county, HHS, national level description
- Add more details to the smoothing process

### CODE REVIEW

File: docs/api/covidcast-signals/quidel.md [hunk 1]
```diff
@@ -68,60 +68,68 @@ $$
...
+We add two kinds of smoothing to the smoothed signals:
+
+##### Temporal Smoothing
+Smoothed estimates are formed by pooling data over time. That is, daily, for
+each location, we first pool all data available in that location over the last 7
+days, and we then recompute everything described in the two subsections above. 
+
+The sample size provided for smoothed signals is different from that for raw signals. 
+The former one is re-definied to be the cumulative sample size in the rolling 7 day 
+window while the latter one is simply the sample size for that corresponding date.
```
Comment (krivard) [inline docs/api/covidcast-signals/quidel.md:99]: Not needed -- "recompute everything described above" already includes sample size as defined here
```suggestion
```

File: docs/api/covidcast-signals/quidel.md [hunk 2]
```diff
@@ -68,60 +68,68 @@ $$
...
 $$
 where $$N_s, X_s$$ are the number of COVID tests and the number of COVID tests
-taken with positive results taken in its home state in the same time period.
+taken with positive results taken in its parent state in the same time period.
 
-**State level**: the states with fewer than 50 tests are discarded. For the
-rest of the states with sufficient samples,
+To be noticed, counties with sample sizes smaller than 50 are merged into megacounties for 
+the raw signals; counties with sample sizes smaller than 25 are merged into megacounties for
+the smoothed signals.
```
Comment (krivard) [inline docs/api/covidcast-signals/quidel.md:125]:
```suggestion
Counties with sample sizes smaller than 50 are merged into megacounties for 
the raw signals; counties with sample sizes smaller than 25 are merged into megacounties for
the smoothed signals.
```

File: docs/api/covidcast-signals/quidel.md [hunk 3]
```diff
@@ -68,60 +68,68 @@ $$
...
+results. 
+
+
+For smoothed signals, after taking the temporal pooling,
+- if $$N \geq 50$$, we still use:
 $$
 p = \frac{100 X}{N}
 $$
-- if $$25 \leq N < 50$$, we lend $$50 - N$$ fake samples from its home state to shrink the
+- if $$25 \leq N < 50$$, we lend $$50 - N$$ fake samples from its parent state to shrink the
```
Comment (krivard) [inline docs/api/covidcast-signals/quidel.md:115]: How do you decide parent state for MSAs/HRRs that cross state boundaries?

File: docs/api/covidcast-signals/quidel.md [hunk 4]
```diff
@@ -68,60 +68,68 @@ $$
...
+taken with positive results taken in its parent state in the same time period.
 
-**State level**: the states with fewer than 50 tests are discarded. For the
-rest of the states with sufficient samples,
+To be noticed, counties with sample sizes smaller than 50 are merged into megacounties for 
+the raw signals; counties with sample sizes smaller than 25 are merged into megacounties for
+the smoothed signals.
 
+**State level, HHS level, Nationa level**: the location with fewer than 50 tests are discarded. 
+For the rest of the location with sufficient samples,
```
Comment (krivard) [inline docs/api/covidcast-signals/quidel.md:128]:
```suggestion
**State level, HHS level, National level**: locations with fewer than 50 tests are discarded. For the remaining locations,
```

**Review by krivard — CHANGES_REQUESTED** [review]
Minor adjustments and a question.

Remember to add the new geo levels to the header as well.

File: docs/api/covidcast-signals/quidel.md (hunk 3 above)
Comment (jingjtang) [reply docs/api/covidcast-signals/quidel.md:112]: The explanation was added. The parent state is defined as the state with the largest proportion of population of that region

File: docs/api/covidcast-signals/quidel.md [hunk 5]
```diff
@@ -119,13 +116,14 @@ p = 100 \left( \frac{N}{50} \frac{X}{N} + \frac{50 - N}{50}  \frac{X_s}{N_s} \ri
 $$
 where $$N_s, X_s$$ are the number of COVID tests and the number of COVID tests
 taken with positive results taken in its parent state in the same time period.
+A parent state is defined as the state with the largest proportion of the population 
+in this county/MSA/HRR.
```
Comment (krivard) [inline docs/api/covidcast-signals/quidel.md:120]: @ryantibs confirming: if an msa or hrr crosses state boundaries, shrinkage only borrows samples from the state of greatest population within that msa/hrr, and not the others

File: docs/api/covidcast-signals/quidel.md [hunk 6]
```diff
@@ -68,60 +68,66 @@ $$
...
-taken with positive results taken in its home state in the same time period.
+taken with positive results taken in its parent state in the same time period.
+A parent state is defined as the state with the largest proportion of the population 
+in this county/MSA/HRR.
 
-**State level**: the states with fewer than 50 tests are discarded. For the
-rest of the states with sufficient samples,
+Counties with sample sizes smaller than 50 are merged into megacounties for 
+the raw signals; counties with sample sizes smaller than 25 are merged into megacounties for
+the smoothed signals.
```
Comment (dshemetov) [inline docs/api/covidcast-signals/quidel.md:124]: I would provide either a short description of megacounties or a link to a description. "For example, the megacounty aggregating counties 20001, 20002, 20003, is 20000."

**Review by dshemetov — APPROVED** [review]
Looks good.

File: docs/api/covidcast-signals/quidel.md (hunk 6 above)
Comment (jingjtang) [reply docs/api/covidcast-signals/quidel.md:124]: Did check the @krivard, we have a general description of the megacounty [here](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_geography.html#small-sample-sizes-and-megacounties), so no more introduction here.

File: docs/api/covidcast-signals/quidel.md (hunk 5 above)
Comment (ryantibs) [reply docs/api/covidcast-signals/quidel.md:120]: Resolved in teams leads. Sorry for the delay!

**Review by krivard — APPROVED** [review]
👍

**Pull request merged**